src/IIIFPresentation/Services/Manifests/Helpers/InterimCanvasPaintingX.cs

```diff
@@ -88,4 +88,8 @@ public static List<CanvasPainting> ConvertInterimCanvasPaintings(
         this List<InterimCanvasPainting>? interimCanvasPaintings) =>
         interimCanvasPaintings?.Where(icp =>
             icp is { SuspectedAssetId: not null, CanvasPaintingType: CanvasPaintingType.Items }).ToList();
+
+    public static IEnumerable<InterimCanvasPainting> OrderInterimCanvasPaintings(
```

Comment:
`CanvasOrder` and `ChoiceOrder` aren't specific to `InterimCanvasPainting`, neither is the logic to `.OrderBy(cp => cp.CanvasOrder).ThenBy(cp => cp.ChoiceOrder)`, those are relevant to `CanvasPainting` too.

If this method was also usable for `CanvasPainting` then it would have more utility and could be used https://github.com/dlcs/iiif-presentation/blob/develop/src/IIIFPresentation/Services/Manifests/ManifestMerger.cs#L101-L102 and https://github.com/dlcs/iiif-presentation/blob/develop/src/IIIFPresentation/API/Converters/ManifestConverter.cs#L173-L174 (or could replace that `ManifestConverter` method entirely).